operations/payments.md

```diff
@@ -190,3 +190,282 @@ Adds a new alternative payment to a specified customer.
+            "Identifier": "",
+            "PaymentType": "Cash",
+            "DataDiscriminator": null,
+            "Data": null
+        }
+    ],
+    "Cursor": "be922eb7-bc5f-4877-b847-1120c0c2acd2"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Payments` | string | required | The filtered payments. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest payment returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of payments. |
+
+#### Payment
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the payment. |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the payment belongs to. |
+| `BillId` | string | optional | Unique identifier of the [Bill](bills.md#bill) the payment is assigned to. |
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](accountingcategories.md#accounting-category) the payment belongs to. |
+| `Amount` | [Amount value](#amount-value) | required | Payment's amount, negative amount represents either rebate or a payment. |
+| `OriginalAmount` | [Amount value](#amount-value) | required | Payment's original amount, negative amount represents either rebate or a payment. Contains the earliest known value in conversion chain. |
+| `Notes` | string | optional | Additional notes. |
+| `SettlementId` | string | optional | Identifier of the settled payment from the external system (ApplePay/GooglePay). | 
+| `ConsumedUtc` | string | optional | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
+| `ClosedUtc` | string | optional | Date and time of the payment bill closure in UTC timezone in ISO 8601 format. |
+| `ChargedUtc` | string | optional | Charged date and time of the payment in UTC timezone in ISO 8601 format. |
+| `CreatedUtc` | string | required | Creation date and time of the payment created in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the payment in UTC timezone in ISO 8601 format. |
+| `AccountingState` | string [Accounting item state](#accounting-item-state) | required | Accounting state of the payment. |
+| `State` | string [Payment state](#payment-state) | required | Payment state of the payment. |
+| `Identifier` | string | optional | Additional unique identifier of the payment. |
+| `PaymentType` | string [Payment type](#payment-type) | required | Payment state of the payment. |
+| `DataDiscriminator` | object [Payment item data](#payment-data-discriminator) | optional | Discriminator pointing to the field in the [Data](#payment-data) field that contains additional information. |
+| `Data` | object [Payment data](#payment-data) | optional | Additional payment data. |
+
+#### Payment data
```

Comment:
If this works like other instances of data discriminators in the API, which I assume it does, then "Payment data" here isn't a real JSON object and it can be removed.